src/ageml/ui.py

```diff
@@ -365,7 +366,11 @@ def load_covariates(self, required=False):
             self.args.covar_name = self.args.covar_name.lower()
             if self.args.covar_name not in df:
                 raise KeyError("Covariate column %s not found in covariates file." % self.args.covar_name)
-     
+
+        # Check if covariate correction mode and set the flag
```

Comment:
what if you just input a covariate file. Does it still do covariate correction? it should do it with cn implicitly

Reply:
`self.flags["covcorr_mode"]` is set to `'cn'`, in the `Interface` initialization, so if the covariate file is provided with no covariate correction mode specified, the mode is `'cn'`.